src/workloads/query/TimeseriesTSBSQuery.yml

```diff
@@ -0,0 +1,1188 @@
+                                    {
+                                      ofLength: 10,
+                                      fromGenerator:
+                                        {
+                                          ^IncDate:
+                                            {
+                                              start: *StartDate,
+                                              step: 2520000,
+                                            },
+                                        },
+                                    },
+                                },
+                              $lte:
+                                {
+                                  ^Cycle:
+                                    {
+                                      ofLength: 10,
+                                      fromGenerator:
+                                        {
+                                          ^IncDate:
+                                            { start: *Hour12, step: 2520000 },
+                                        },
+                                    },
+                                },
+                            },
+                          usage_user: { $gt: 90 }, # 90 is hard-coded in TSBS.
+                        },
+                    },
+                    { $set: { tags: "$tags.hostname" } },
+                  ]
+
+  - Name: lastpoint
+    Type: CrudActor
+    Threads: 1
+    Phases:
+      OnlyActiveInPhases:
+        Active: [15]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  [
+                    { $sort: { "tags.hostname": 1, "time": -1 } },
+                    {
+                      $group:
+                        {
+                          _id: { "hostname": "$tags.hostname" },
+                          time: { $first: "$time" },
+                          measurement: { $first: "$measurement" },
+                          usage_softirq: { $first: "$usage_softirq" },
+                          musage_steal: { $first: "$usage_steal" },
+                          usage_guest: { $first: "$usage_guest" },
+                          usage_user: { $first: "$usage_user" },
+                          usage_system: { $first: "$usage_system" },
+                          usage_iowait: { $first: "$usage_iowait" },
+                          usage_irq: { $first: "$usage_irq" },
+                          usage_nice: { $first: "$usage_nice" },
+                          usage_idle: { $first: "$usage_idle" },
+                          usage_guest_nice: { $first: "$usage_guest_nice" },
+                        },
+                    },
+                  ]
+
+  # The queries below are not from tsbs_query, but are typical user queries.
```

Comment:
I'm not sure about the "typical" claim. I'd replace it with a description of what kind of optimizations/behaviours we are trying to validate with these pipelines. For example, your non-selective query essentially stresses unpacking, given you are adding a separate workload just for that, is it worth adding it here? 

For the other two, I presume the goal is to test bucket-targeting, right? What are our expectations for the distributions of the measurements across the buckets? In the tsbs dataset I'm seeing ~161K total buckets and ~31K buckets that match a filter like `{"control.min.usage_nice": {$lte: 80}, "control.max.usage_nice": {$gte: 80}}` (which gets inserted for `{$match: {usage_nice: 80}}`). That is, almost 20% of the buckets, and that's for a point-query. Extend the query to a range and you might end up with having to open half of the buckets if not more, rendering the test to be very slow and not really giving you good signal on bucket targeting.